src/core/src/FabricGeometry.cpp

```diff
@@ -158,7 +153,7 @@ void FabricGeometry::initialize() {
     }
 
     if (hasVertexColors) {
-        attributes.addAttribute(FabricTypes::primvars_vertexColor, FabricTokens::primvars_vertexColor);
```

Comment:
Is the vertexColor attribute never going to be used going forward for anything, such as styling?

Reply:
Ah, I see. Got it. Nevermind.

Reply:
Yeah it's basically just a rename from `vertexColor` to `displayColor`. Trying to be more consistent with USD conventions.